lib/vmdb/settings/walker.rb

```diff
@@ -1,8 +1,8 @@
 module Vmdb
-  class Settings
-    module Walker
-      PASSWORD_FIELDS = %i(bind_pwd password amazon_secret ssh_key_data ssh_key_unlock become_password vault_password security_token).to_set.freeze
+  module SettingsWalker
```

Comment:
> I think the diff will be fine if the git mv happens in a commit with no other changes.

Yeah, I think I can do this in two commits in such a way that it would achieve that, and also not have a commit that breaks the build (a pedantic thing I for some reason strive for.

> Based on the module name, I would expect the file to be `lib/vmdb/settings_walker.rb`

Yeah, I will go with that.  Hopefully no one else has a problem with that name/filename.